codespell_lib/_codespell.py

```diff
@@ -566,6 +579,9 @@ def parse_options(
             for toml_file in toml_files:
                 with open(toml_file, "rb") as f:
                     data = tomllib.load(f).get("tool", {})
+                for tool, tool_dict in data.items():
+                    if tool == "codespell":  # only "codespell" for performance
+                        data[tool] = _toml_to_parseconfig(tool_dict)
```

Comment:
... then this would just be
```suggestion
                if "codespell" in data:  # only "codespell" for performance
                    _toml_to_parseconfig(data["codespell"])
```

Reply:
This one applies seamlessly.

Reply:
Applied.